docs/src/dedicated-gen-3/overview.md

```diff
@@ -0,0 +1,51 @@
+
+Dedicated Generation 3 works nearly identically to Platform.sh environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 3?
+
+Our existing [dedicated infrastructure](/dedicated/overview) ensures increased uptime and availability for your applications and services, but configuration or application topology changes have to go through our Customer Success team, as the VMs are provisioned by us. Generation 3 gives you both the high availability from our Dedicated offer combined with the self-service flexibility and features of Platform.sh Professional. 
+
+This means that you are able to edit your configuration yourself - in your `services.yaml`, `routes.yaml`, and `.platform.app.yaml` files - and then see those changes reflected in your Dedicated Generation 3 environments on every push without opening a ticket. 
+
+## Features
+
+* A minimum of three virtual machine cluster is associated with your default (production) environment, and also optionally with a staging environment. 
+* Configuration changes on development environments (through your `services.yaml`, `routes.yaml`, and `.platform.app.yaml` files) are reflected on these Dedicated Gen 3 clusters when you merge them. It is not necessary to open a support ticket to change production infrastructure like it is on Dedicated. 
+* You can deploy your application in any of our [supported cloud providers](/development/faq.md#which-geographic-zones-does-platformsh-cover).
+
+## Differences from the Grid
+
+Although Dedication Generation 3 adds plenty of features to your existing Grid applications, there are a few differences and limitations to be aware of when considering an upgrade. 
+
+### Available services
+
+In our [services documentation](/configuration/services) you will notice that not every service is available on Dedicated Generation 3 that is available on the Grid, nor is every version. The table below shows the currently available services and their versions for Dedicated Generation 3. 
+
+{{< gen-3-services >}}
+
+### Local mounts
+
+Because you get a redundant infrastructure, note that local mounts will be local to each Virtual Machine. Since you can't know which VM is going to handle a specific request, you also don't have a guarantee regarding which local mount is going to be used. Whether you actually want to use a local mount or in fact need to set up a network storage mount depends on your specific use-case.
+
+## Setting up
+
+If you are running a non-Dedicated Platform.sh project (a Professional or non-Dedicated Enterprise plan), migrating it to Dedicated Generation 3 can be done by opening a support ticket to request it. At the moment, Dedicated projects cannot be migrated to Dedicated Generation 3, but support for this type of migration will soon be available. 
```

Comment:
This section feels a bit mixed up. 

To move from Pro to Dedicated Generation 3, customers need to go through sales. 

Upgrades from Dedicated Generation 2 to Dedicated Generation 3 can and should go through a support ticket once available. 

@nickbennison  which of these options are currently available?

Reply:
We can probably send foks to https://platform.sh/contact/ for upgrades to Dedicated

Reply:
It is.  I chatted with @chadwcarlson about this section already.  So to clarify...

- Grid to ded gen3 migrations are not available and not on the roadmap at the moment.
- Ded gen2 to ded gen 3 migrations work is underway, but not ready for production yet.

Contact us was stripped out for now based on a call we had with @OriPekelman.  Seemed to make more sense to have people reach out to us when we're closer to be able to do the migrations, but that can be changed...